packages/terra-tabs/src/Tabs.module.scss

```diff
@@ -96,6 +96,7 @@
     span {
       white-space: normal;
     }
+    /* stylelint-enable selector-max-compound-selectors */
```

Comment:
oops, I missed that we use a `disable-next-line`, so we don't need to enable the rule

Reply:
True, this comment can be removed or we should update the comment on line 95 to remove the `-next-line` part.